MAUI/Parallax-View/Getting-started.md

```diff
@@ -0,0 +1,271 @@
+    using Microsoft.Maui.Hosting;
+    using Microsoft.Maui.Controls.Compatibility;
+    using Microsoft.Maui.Controls.Hosting;
+    using Microsoft.Maui.Controls.Xaml;
+    using Syncfusion.Maui.Core.Hosting;
+
+    namespace ParallaxView_GettingStarted
+    {
+        public static class MauiProgram
+        {
+            public static MauiApp CreateMauiApp()
+            {
+                var builder = MauiApp.CreateBuilder();
+                builder
+                .UseMauiApp<App>()
+                .ConfigureSyncfusionCore()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                });
+
+                return builder.Build();
+            }
+        }
+    }
+
+{% endhighlight %}
+
+
+## Add content to the parallax view
+
+### Content
+
+The [`Content`]() represents the background view of a parallax view. You can set any kind of view to the [`Content`]() property such as Image and StackLayout.
+
+The following code sample demonstrates how to set the content property to the parallax view.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+   <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:ParallaxSample"
+             xmlns:parallax="clr-namespace:Syncfusion.Maui.ParallaxView;assembly=Syncfusion.Maui.ParallaxView"
+             x:Class="ParallaxView_GettingStarted.MainPage">
+      
+    <ContentPage.Content>
+        <Grid>            
+        <parallax:SfParallaxView x:Name="parallaxview">
+            <parallax:SfParallaxView.Content>
+                <Image Source="{Binding Image}" BackgroundColor="Transparent" HorizontalOptions="Fill" VerticalOptions="Fill" Aspect="AspectFill" />
+            </parallax:SfParallaxView.Content>
+        </parallax:SfParallaxView>       
+        </Grid>
+    </ContentPage.Content>
+	
+    </ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.ParallaxView;
+
+    namespace ParallaxView_GettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                BindingContext = new ParallaxViewModel();
+            }
+        }
+
+        public class ParallaxViewModel
+        {
+            public ImageSource Image { get; set; }
+
+            public ParallaxViewModel()
+            {
+                Image = ImageSource.FromResource("ParallaxView_GettingStarted.ParallaxGuitar1.png", typeof(MainPage).GetTypeInfo().Assembly);
+            }
+        }  
+    }  
+
+{% endhighlight %}
+{% endtabs %}
+
+## Bind source to the parallax view
+
+### Source
+
+The [`Source`]() represents the foreground view of the parallax view. The value of [`Source`]() should be a scrollable content or the view which implements [IParallaxView]() interface.
```

Comment:
The [`Source`]() represents the foreground view of the parallax view. The value of the [`Source`]() should be a scrollable content or the view which implements the [IParallaxView]() interface.